docs/modules/ROOT/pages/usage.adoc

```diff
@@ -4,29 +4,29 @@ After installation, the CRD for this operator must be created:
 
     kubectl apply -f /etc/stackable/kafka-operator/crd/kafkacluster.crd.yaml
 
-To create an Apache Kafka (v2.8.0) cluster named "simple" with Prometheus metrics exposed on port 9606:
+To create an Apache Kafka (v2.8.0) cluster named `simple-kafka` assuming that you already have a Zookeeper cluster named `simple-zk`:
```

Comment:
There is only `2.8.1` image in the repo afaik. (Dont forget to adapt in the examples too).